src/fmu/dataio/providers/_fmu.py

```diff
@@ -104,7 +104,7 @@ class FmuProvider:
 
     model: dict | None = None
     fmu_context: FmuContext = FmuContext.REALIZATION
-    include_ertjobs: bool = True
+    include_ertjobs: bool = False
```

Comment:
hehe nice catch.. yes was going to make a comment regarding this 🙂. I saw that in `ExportData` this is defaulted to `False` by an class variable. So I figured changing the default here were appropriate, and then I did not expose this as an argument when initializing the `FmuProvider` in the `ExportPreprocessedData`.
If I am not mistaken I think @perolavsvendsen mentioned that storing of the jobs.json per data might not be the desired way in the future? Am I correct @perolavsvendsen?

Reply:
I think we stopped including the `jobs.json` quite quickly, yes. It becomes very, very large in some cases. Spamming the metadata. Rather include it in the future if there is a (more) clear need for it.

Reply:
In case that wasn't very clear: As far as I am concerned, we can stop supporting this completely. Not sure if we have other dependencies on _reading_ it, but I am quite certain we can completely stop support dumping it to metadata.

Reply:
> In case that wasn't very clear: As far as I am concerned, we can stop supporting this completely. Not sure if we have other dependencies on _reading_ it, but I am quite certain we can completely stop support dumping it to metadata.

I like the sound of that 👍

Reply:
created issue #664